tensorflow/lite/micro/kernels/svdf.cc

```diff
@@ -48,6 +48,7 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       tflite::micro::GetEvalInput(context, node, kSvdfWeightsFeatureTensor);
   const TfLiteEvalTensor* weights_time =
       tflite::micro::GetEvalInput(context, node, kSvdfWeightsTimeTensor);
+  // TODO(tflite-micro#1751) account for optional bias tensor
```

Comment:
let's replace all the TODOs with `TODO(#1751):`

We have some internal checks that enforce a specific consistent format for the TODO.

Reply:
done